src/clr-core/badge/badge.element.scss

```diff
@@ -1,7 +1,9 @@
 @import './../styles/mixins/mixins';
-@import './properties';
 
 :host {
+  --background: #{$clr-color-neutral-600};
```

Comment:
I feel pretty strongly that we should use `var(--clr-color-neutral-600)` in these cases instead of forcing users to rely on SASS variables that they won't be able to override in our next major version...

Reply:
Updated to use color vars with sass fallbacks.